packages/cfd/src/Containers/mt5-mobile-download-option.scss

```diff
@@ -0,0 +1,32 @@
+.mt5-download-container {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    margin: 2.4rem;
+
+    &--option {
+        height: 34px;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding: 8px;
+        gap: 8px;
+        border-radius: 8px;
+        background-color: #f2f3f4;
+        p {
+            flex: 1;
+        }
+    }
+
+    .blue {
+        background-color: #0364b9;
+        p {
+            color: #fff;
```

Comment:
use our variables for coloring to work properly for the dark theme as well. Make sure you check the dark them for your changes